services/export/shared/helpers/s3.js

```diff
@@ -177,6 +177,25 @@ const exists = async (fileName) => new Promise((resolve) => {
 		.catch(() => resolve(false));
 });
 
+const isFile = async (filePath) => {
+	const fileExists = await exists(filePath);
+	if (!fileExists) {
+		return false;
+	}
+
+	const stat = await new Promise((resolve) => {
+		minioClient.statObject(AWS_S3_BUCKET_NAME, filePath, (err, stats) => {
+			if (err) {
+				resolve(0);
```

Comment:
Shouldn't the return value be an object? `stat.size` below would throw an error. Alternatively, we can directly resolve the boolean value within the callback if..else ladder.
```suggestion
				resolve({ size: 0 });
```